### ISSUE TITLE
When detecting shell scripts take into consideration that the changed file could be a symlink :link: 

### ISSUE THREAD

**Issue by jamacku** [redhat-plumbers-in-action/differential-shellcheck#111]
### Type of issue

Bug Report

### Description

In #110 I have added symlink, and in the output of Differential ShellCheck, I saw the following [logs](https://github.com/redhat-plumbers-in-action/differential-shellcheck/runs/8228703725?check_suite_focus=true#step:4:1103):

```sh
head: error reading 'src': Is a directory
grep: src: Is a directory
grep: src: Is a directory
grep: src: Is a directory
```

### Describe the solution you'd like

Skip symlinks and don't report anything unless in DEBUG mode.
